test/src/test/java/org/corfudb/runtime/collections/PersistentCorfuTableTest.java

```diff
@@ -48,367 +55,266 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.fail;
+import static org.corfudb.runtime.ExampleSchemas.SportsProfessional;
+import static org.corfudb.runtime.ExampleSchemas.TrainingPlan;
+import static org.corfudb.runtime.view.ObjectsView.ObjectID;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.doReturn;
 import static org.mockito.Mockito.doThrow;
 import static org.mockito.Mockito.reset;
 import static org.mockito.Mockito.spy;
 
-@SuppressWarnings("checkstyle:magicnumber")
 @RunWith(MockitoJUnitRunner.class)
 public class PersistentCorfuTableTest extends AbstractViewTest {
```

Comment:
Is the intention for these to run with DiskBacked tables too? Or will these be combined with PersistedCorfuTable tests in a subsequent PR?

Reply:
Either way is ok, I think we will need to combine two tests in the end, just to avoid repeating tests and have all the tests in one place